src/runtime-rs/src/shim/Cargo.toml

```diff
@@ -30,4 +30,4 @@ slog-scope = "4.4"
 slog-stdlog = "4.1"
 nix = "0.22"
 serde_json = "1.0"
-toml = "0.5"
\ No newline at end of file
+toml = "0.5"
```

Comment:
This change should be folded into the original commit.

Reply:
Fixed in the second revision